src/index.html

```diff
@@ -17,6 +17,88 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo_img"
+          src="./images/logo.png"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Since `index.html` is not meant to be an anchor link but a URL, this is fine. However, if it were an anchor link, it should start with `#` and not contain spaces.

Reply:
I've fixed the bugs, look at the new commit.